source/languages/en/riak/dev/using/conflict-resolution/csharp.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Conflict Resolution: CSharp"
+project: riak
+version: 2.0.0+
+document: tutorials
+audience: intermediate
+keywords: [developers, conflict-resolution, csharp]
+---
+
+For reasons explained in the [[introduction to conflict resolution|Conflict Resolution]], we strongly recommend adopting a conflict resolution strategy that requires applications to resolve siblings according to use-case-specific criteria. Here, we'll provide a brief guide to conflict resolution using the official [Riak .NET client](https://github.com/basho-labs/riak-dotnet-client).
```

Comment:
This looks to be the only new file that doesn't wrap lines. I'd like to go back and break these paragraphs up, but it's not hugely important.